src/main/kotlin/pl/put/smartgarden/domain/user/UserAuthService.kt

```diff
@@ -54,14 +51,20 @@ class AuthService(
         return verificationToken.user
     }
 
+    /**
+     * Checks if token is valid and returns logged in user or throws appropriate exception.
+     */
     fun getUserFromJWToken(token: String): User {
```

Comment:
Moze tu lepeij skorzystac z metody w chyba securiutyService, ktora zwraca Ci id z tokena?

Reply:
Nie jestem pewien czy to powinno w ogóle być w SecurityService 🤔  bo w końcu tam miały być takie generyczne rzeczy zarówno do autoryzacji użytkownika jak i device.